### ISSUE TITLE
`one-var": [2, { initialized: "never" }]` raised when multiple declarations exist in for loop

### ISSUE THREAD

**Issue by feross** [eslint/eslint#2505]
This is related to #2342.

I want to use the rule `"one-var": [2, { initialized: "never" }]`. It mostly works correctly, except it complains about:

```js
var arr = [1, 2, 3];
for (var i = 0, len = arr.length; i < len; i++) {
  // ...
}
```

With this error: `Split initialized 'var' declarations into multiple statements.`

Is there some way to ignore statements inside for loops? I don't mind multiple declarations in there, but I do want to prevent this:

```js
var i = 0, j = 5, k = 99;
```

**Comment by nzakas** [comment]
We should probably just never warn about that for for loops.

**Comment by feross** [comment]
Agreed.

**Comment by btmills** [comment]
Working on this.

**Comment by Mouvedia** [comment]
@nzakas Could you add a ¶ about the fact that the rule ignores for loops on that [page](http://eslint.org/docs/rules/one-var)?
